src/components/Pool/LoadingList/LoadingCard/index.tsx

```diff
@@ -17,6 +17,11 @@ const SizedCard = styled(DarkCard)<{ isMobile: boolean }>`
    height:128px;
        overflow: hidden;
   `};
+  ::before {
+    background-blend-mode: normal, overlay, normal;
+    background: linear-gradient(143.3deg, rgba(46, 23, 242, 0.5) -185.11%, rgba(46, 23, 242, 0) 49.63%),
+      linear-gradient(113.18deg, rgba(255, 255, 255, 0.15) -0.1%, rgba(0, 0, 0, 0) 98.9%), #171621;
```

Comment:
I see the `#171621` color in several components. Should we add it to the theme?

Reply:
Yes, add it as another `bg`. As theres is no drakMode for this color I think that this should work for both modes for now.